include/graphblas/bsp1d/init.hpp

```diff
@@ -71,298 +75,346 @@ namespace grb {
 		 */
 		class BSP1D_Data {
```

Comment:
is this class copy-constructible/assignable? is it move-constructible/assignable? or is it some sort of singleton? for safety, the various copy constructor/assignment should be properly implemented (probably just `= delete`).

Reply:
will fix